tools/docker-network/config.docker.json

```diff
@@ -70,7 +70,8 @@
   "pow": {
     "difficulty": 2,
     "numThreads": 1,
-    "timeout": "10s"
+    "timeout": "10s",
```

Comment:
lets use 60s as default

Reply:
here 10s is fine as it is for the docker-network where we set the difficulty to be only 2